.travis.yml

```diff
@@ -1,9 +1,7 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
   - "0.12"
-  - "4.2"
+  - "4"
```

Comment:
@fatso83 it seems that when you set `4` in the Travis matrix, Travis will use the latest version of Node.js 4.

Here is an example on `gulp-template`, which has the [following Travis matrix](https://github.com/sindresorhus/gulp-template/blob/master/.travis.yml).
You'll see that [gulp-template latest job on Travis for Node 4](https://travis-ci.org/sindresorhus/gulp-template/jobs/100388923) used the latest version of Node.js 4 (4.2.4) at the time it was run (4.3.0 was not available yet when that job ran). I verified this with other modules, and this behavior is confirmed.

So to *always use the latest LTS release for this major version*, I would just set a major version in Travis matrix, like`4` and/or `5`, but no minor versions like `4.3`